fe/fe-core/src/test/java/com/starrocks/persist/CreateCatalogTest.java

```diff
@@ -0,0 +1,46 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.persist;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.*;
+import java.util.HashMap;
+
+public class CreateCatalogTest {
+
+    private String fileName = "./CreateCatalogTest";
+
+    @After
+    public void tearDownCreate() {
+        File file = new File(fileName);
+        file.delete();
```

Comment:
handle exception in case of unit test failure

Reply:
> handle exception in case of unit test failure

done